polaris/viz/planar.py

```diff
@@ -167,6 +167,8 @@ def plot_horiz_field(ds_mesh, field, out_file_name=None,  # noqa: C901
         pcolor_kwargs['norm'] = LogNorm(
             vmin=max(1e-10, vmin), vmax=vmax, clip=False
         )
```

Comment:
```suggestion
        pcolor_kwargs['norm'] = LogNorm(
            vmin=max(1e-10, vmin) if vmin else 1e-10, vmax=vmax, clip=False
        )
```

Reply:
@andrewdnolan, the problem is that `vmin = 0.0` will evaluate to `False`.  That happens to work here but it's best to do:
```suggestion
        pcolor_kwargs['norm'] = LogNorm(
            vmin=max(1e-10, vmin) if vmin is not None else 1e-10, vmax=vmax, clip=False
        )
```
At this point, I think that's needlessly hard to read so I'll write it differently.

Reply:
Good catch. Sounds good.